src/indieweb_utils/indieauth/profile.py

```diff
@@ -20,6 +20,26 @@ class Profile:
 
 
 def get_profile(me: str) -> Profile:
+    """
+    Get a profile from a url.
+
+    :param me: The url to get the profile from.
+    :type me: str
+    :return: The profile.
+    :rtype: Profile
+
+    Example:
+
+    .. code-block:: python
+
+        import indieweb_utils
+
+        me = "https://jamesg.blog"
+
+        profile = indieweb_utils.get_profile(me)
+
+        print(profile.name) # James
```

Comment:
What if instead of doing print examples like that we just did assert for the expected results?

```
assert profile.email == "james@jamesg.blog"
assert profile.name == "James"
assert profile.photo == "https://jamesg.blog/me.jpg"
assert profile.url == "https://jamesg.blog